tags/pages/action.tag.html

```diff
@@ -1,8 +1,8 @@
 <page-action>
   <article class=block>
     <h1>{ this.action.name }</h1>
-    <section class=description>
-      { this.action.description }
+    <section class=description if={ this.action.description }>
+      <marked text={ this.action.description }></marked>
```

Comment:
I'd have prefered `<marked>{ this.action.description }</marked>` but it's buggy :(